dev/build_release.py

```diff
@@ -236,14 +236,24 @@ def start_container_build(self, name):
                 project=self.__options.gcb_project)
       )
     elif self.__options.container_builder == 'docker':
-      return BackgroundProcess.spawn(
-          'Build/publishing container image for {name} with Docker...'.format(
+      if name == 'spinnaker-monitoring':
```

Comment:
It looks like both these are the same except for the path to the docker.yml file. Could you just have if/else on that path then run through a single line where the command is to emphasize how they are the same/different?